lib/features/keyboard/Keyboard.js

```diff
@@ -125,7 +125,7 @@ Keyboard.prototype._isEventIgnored = function(event) {
 /**
  * Bind keyboard events to the given DOM node.
  *
- * @param {EventTarget} node
+ * @param {undefined} node
  */
 Keyboard.prototype.bind = function(node) {
```

Comment:
We should rather fix the method signature, not require any argument, and check whether an argument is provided via `arguments[0]`. This change looks odd.

```suggestion
Keyboard.prototype.bind = function() {
```

Reply:
Much better solution, thank you! 👍 (I wasn't sure if I could change the signature tbh)